mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
+    if (!order) {
+      return res.status(404).json({ message: "Order does not exist" });
```

Comment:
The error message should be consistent throughout the application. In this case, the error message should be wrapped in an `error` key, similar to other error responses in the code. It should be `return res.status(404).json({ error: "Order does not exist" });` instead.